src/main/js/src/containers/ExamDates/ExamDates.js

```diff
@@ -372,20 +370,21 @@ class ExamDates extends Component {
                   data-cy={`exam-dates-add-eval-button-${e.exam_date}`}
                   className={classes.EditButton}
                   style={{ width: '90%' }}
-                  onClick={() => this.showAddEvaluationPeriodHandler(e)}
+                  onClick={() => this.showAddEvaluationPeriodModal(e)}
                 >
-                  {t('examDates.add.evaluation.period')}
+                  {t('examDates.evaluation.period')}
                 </button>
               )}
 
-              <button
-                data-cy={`exam-dates-edit-button-${e.exam_date}`}
-                disabled={!canEditExamDate(e)}
-                className={classes.EditButton}
-                onClick={() => this.showEditExamDateHandler(e)}
-              >
-                {t('common.edit')}
-              </button>
+              {editShown(e) ? (
+                <button
+                  data-cy={`exam-dates-edit-button-${e.exam_date}`}
+                  className={classes.EditButton}
```

Comment:
Tässä samassa voisi muuten vaihtaa `classes.EditButton` määritellyn `height: 2rem` -> `height: 3rem`. Ei tietenkään mikään täydellinen asetus sekään, mutta kuitenkin varmasti parannus nykytilaan. Jos `height: max(2rem, fit-content)` olisi varmaan paras ratkaisu, mikäli ko. syntaksi olisi sallittu.

Reply:
Aa, menin tosiaan luomaan sen uuden `Tarkistusarviointi` nimellä olevan avaimen jota se "Lisää tarkistusarviointi käyttää". Tääkin toinen tapa millä tätä vois lähestyä, mutta nyt ei tarvitse korkeussäätöä muuttaa jos menee taas tässä jo toteutetulla ratkaisulla.

Reply:
Ok, enpä tajunnut että se avain oli tätä buttonia varten. Sotii hieman omaa intuitiota vastaan että buttonin call to action on substantiivi eikä aktiivinen kehote.

Reply:
3rem mitalla saa tuon napin tosiaan näkymään kivasti, muu rivin teksti ei vaan tuohon kauhean näppärästi suoraan sovellu. Tai mm. "Muokkaa" nappula 3rem korkeudella menee tällä hetkellä pohjastaan ikävästi lähelle seuraavaa taulukon riviä ja noita nappuloita on hieman hankalaa katsoa että mihin riviin liittyvät. Pitäis värjätä myös rivit tms. melkeinpä jos vaihtais 3rem tai tehdä jotain muuta.

Reply:
Mun mielestä toi 3rem kauttaaltaan näytti ihan ok. Voi olla että devisetupilla tulee hieman eri vaikutelma, mutta ainakin tuotantodatalla rivit ei tuntunut mun makuun liian ahtailta.

En tiedä onko sekään kiva että nuo eri napit on eri korkuisia.

Reply:
Tässä kun poistin tuon checkboxin edestä ja muutin `grid-template-columns` css:ää niin kieli-tasot menevät kätevästi omille riveilleen ja ainakin yhden kielen tapauksessa tuo nappula nyt määrittää rivin korkeuden ja teksti jää sen ylälaidan kohdalle. Jos useampi kieli-taso valittuna tutkintopäivälle, niin näkyy toimivan varsin hyvin.

Tässä uusimmassa commitissa ei muuten jostain syystä näytä menneiden tilaisuuksien tasoja. Tsekkaan tuon.

Reply:
Okei, elikkäs bugi jonka luulin olevan läsnä johtuu näemmä vain siitä että `examDates.json` tiedostossa ei kaikille menneiden tutkintopäivien kielille tasoa määritelty.